build-index.sh

```diff
@@ -117,8 +117,8 @@ elif [[ "$format" = "md" ]]; then
         echo
     }
     function table_i() {
-        echo "| Draft |     |     |     |     |     |     |"
-        echo "| ----- | --- | --- | --- | --- | --- | --- |"
+        echo "| Draft |     |     |     |     |"
+        echo "| ----- | --- | --- | --- | --- |"
```

Comment:
This just highlights a problem...

The main branch has this many, but other branches have one more.

I've been considering dropping the issue link for branches other than the main one.  That might make this easier.  WDYT?

Reply:
I think dropping `issues` as well as `datatracker` makes lot of sense. One might even argue that `diff with last submission` isn't relevant to branches...